qt_gui/src/qt_gui/plugin_handler.py

```diff
@@ -253,7 +253,8 @@ def _update_dock_widget_features(self, dock_widget):
     def _update_title_bar(self, dock_widget, hide_help=False, hide_reload=False):
         title_bar = dock_widget.titleBarWidget()
         if title_bar is None:
-            title_bar = DockWidgetTitleBar(dock_widget, self._application_context.qtgui_path)
+            title_bar = DockWidgetTitleBar(dock_widget, self._application_context.qtgui_path,
+                                           self._application_context.options.hide_title)
```

Comment:
Please pass the new argument explicitly as a keyword argument (`, hide_title=self._application_context.options.hide_title`).

Nitpick: please add a newline after the `(` to avoid the aligned indentation and instead indent the wrapped line with standard 4 spaces.